ionic-app/store/actions.js

```diff
@@ -0,0 +1,32 @@
+import Store from '.';
+
+export const setMenuOpen = open => {
+  Store.update(s => {
+    s.menuOpen = open;
+  });
+};
+
+export const setNotificationsOpen = open => {
+  Store.update(s => {
+    s.notificationsOpen = open;
+  });
+};
+
+export const setSettings = settings => {
+  Store.update(s => {
+    s.settings = settings;
+  });
+};
+
+// App-specific actions
+
+export const setDone = (list, item, done) => {
+  Store.update((s, o) => {
+    const listIndex = o.lists.findIndex(l => l === list);
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/toost-net/toost-cloud/pullRequest?prid=8079923)
```suggestion
    const listIndex = o.lists.findIndex((l) => l === list);
```